config/builds.js

```diff
@@ -37,7 +37,9 @@ const builds = buildConfigs
   })
   .map(buildConfig => {
     const name = buildConfig.name || '';
-    const env = buildConfig.env || {};
+    const env = buildConfig.env || {
+      SKU_TENANT: args.tenant
```

Comment:
This is how I did it in my spike, but I'm not sure it will let anything with a buildConfig pick up the tenant variable.  Wasn't sure if I wanted to make it a compulsory pickup by spreading `buildConfig.env` into an object with `SKU_TENANT` and exporting that though.